website/docs/d/scc_posture_latest_scans.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_latest_scans"
+description: |-
+  Get information about list_latest_scans
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_latest_scans
+
+Provides a read-only data source for list_latest_scans. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
Provides a read-only data source for list_latest_scans. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.

as

Review information about the security and compliance center posture latest scans. For more information, about  ..., see []().